chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/surveillance/SurveillanceProcessType.java

```diff
@@ -0,0 +1,65 @@
+package gov.healthit.chpl.domain.surveillance;
```

Comment:
Are we actually including this data in the XML download?

Reply:
No! good catch! Removed annotations.